qualification/report/katdoc.sty

```diff
@@ -103,147 +123,120 @@
+                                                                                             & 2 Fir Street,          & Krugersdorp,      & Carnarvon,                     \\
+                                                                                             & Observatory,           & 1740,             & 8925,                          \\
+                                                                                             & Cape Town,             & South Africa      & South Africa                   \\
+                                                                                             & 7925,                  &                   &                                \\
+                                                                                             & South Africa           &                   &                                \\
+      \hline
+      Tel.                                                                                   & +27 21 506 7300        & +27 12 301 3100   & +27 21 506 7300                \\
+      \hline
+      Fax                                                                                    & +27 21 506 7375        & +27 12 301 3300   & +27 86 538 6836                \\
+      \hline
+      \small
+      Website                                                                                & www.sarao.ac.za        & www.hartrao.ac.za & www.sarao.ac.za                \\
+      \hline
+    \end{tabularx}
+    \vspace{1cm}
     \normalsize
 
     % reset table cell padding
     \renewcommand{\arraystretch}{1.0}
-
-  \end{titlepage}
-
+  \end{center}
 }
 
-
-
-
 % General Page Header and Footer
-
-\clearpage
-
-\pagestyle{fancy}
-
-\fancyhead{} % clear all header fields
-\fancyhead[C]{\docClassification}
-\fancyhead[R]{\docId \\ Revision: \docRevision}
-
-\fancyfoot{} % clear all footer fields
-\fancyfoot[L]{\docDate}
-\fancyfoot[C]{\docClassification}
-\fancyfoot[R]{Page \thepage\ of \pageref{LastPage}}
-
-\renewcommand{\headrulewidth}{0.4pt}
-\renewcommand{\footrulewidth}{0.4pt}
-
-% Abbreviations
-
-\newcommand{\abbrev}[2]{
-  #1 & #2\\
-  \hline
-}
-
-\newcommand{\listofabbreviations}{
-  \section*{Glossary}
-
-  \begin{tabular}{|p{0.1\textwidth}|p{0.9\textwidth}|}
-    \hline
-    \abbreviations
-  \end{tabular}
+\fancypagestyle{normal}{%
+  \fancyhead{} % clear all header fields
+  \fancyhead[C]{\docClassification}
+  \fancyhead[R]{\docId \\ Revision: \docRevision}
+
+  \fancyfoot{} % clear all footer fields
+  \fancyfoot[L]{\docDate}
+  \fancyfoot[C]{\docClassification}
+  % The * is to make hyperref not link this one.
+  \fancyfoot[R]{Page \thepage\ of \pageref*{LastPage}}
+
+  \renewcommand{\headrulewidth}{0.4pt}
+  \renewcommand{\footrulewidth}{0.4pt}
 }
 
 % The entire beginning part of the document
 
-\newcommand{\makekatdocbeginning}{
+\newcommand{\makekatdocbeginning}{%
+  \pagenumbering{roman}
```

Comment:
Do I understand correctly that you're changing the preamble from Arabic to Roman numbering? SE/config management might not like that if it's not consistent with their templates (I recall Simon once getting all but one signature on a document, with someone insisting it had to be redone because the page numbering wasn't done right).

It also means that the "Page N or M" footer will only be counting the number of Roman-numbered pages, not the number of pages in the document.

Reply:
This matches what is currently done in the design doc, which was an issue that @tyronevb flagged. So I'm presuming that this is what SE wants.

Reply:
Ok, if it's what they want I'm happy with it.

Reply:
I see what you mean. @tyronevb, can you comment on what the SE style guide says?